bundles/org.openhab.binding.modbus.lambda/src/main/resources/OH-INF/thing/lambda-heatpump-types.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="modbus"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="lambda-heatpump">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="tcp"/>
+		</supported-bridge-type-refs>
+		<label>Lambda Heat Pump</label>
+		<description>Lambda Heat Pump connected through modbus</description>
+
+		<channel-groups>
+			<channel-group id="generalAmbient" typeId="general-ambient"/>
+			<channel-group id="generalEManager" typeId="general-emanager"/>
+			<channel-group id="Heatpump1" typeId="heatpump1"/>
+			<channel-group id="Boiler1" typeId="boiler1"/>
+			<channel-group id="Buffer1" typeId="buffer1"/>
+			<channel-group id="HeatingCircuit1" typeId="heatingcircuit1"/>
```

Comment:
Please adjust to the naming convention `lower-case-hypen`
Both the `id` as `typeId`

For reference: https://www.openhab.org/docs/developer/guidelines.html#naming-convention